src/types/index.ts

```diff
@@ -0,0 +1,58 @@
+export interface User {
```

Comment:
Updating the types for user, login, register, appointment, appointment data for dashboard, and the authentication response respectively